db/migrate/20220428234145_add_users_ref_to_posts.rb

```diff
@@ -0,0 +1,5 @@
+class AddUsersRefToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :user, null: false, foreign_key: true
```

Comment:
- [ ] <b>REQUIRED:</b> Now the foreign key of the user in the posts table will be named `user_id` and the project requires to make sure that the foreign key for the user in  posts table is named `author_id` instead of `user_id`. Kindly fix it.

Reply:
Hi thanks for the review, Kindly ask you to share with me the link of how to change this because following the milestone resources the teach to do it like that